<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>11</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Notepad Application</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QTextEdit" name="textEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_as"/>
    <addaction name="separator"/>
    <addaction name="actionPrint"/>
    <addaction name="actionPrint_Preview"/>
    <addaction name="actionExport_PDF"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionUndo"/>
    <addaction name="actionRedo"/>
    <addaction name="separator"/>
    <addaction name="actionCut"/>
    <addaction name="actionCopy"/>
    <addaction name="actionPaste"/>
   </widget>
   <widget class="QMenu" name="menuFormat">
    <property name="title">
     <string>Format</string>
    </property>
    <addaction name="actionBold"/>
    <addaction name="actionItalic"/>
    <addaction name="actionUnderline"/>
    <addaction name="separator"/>
    <addaction name="actionLeft"/>
    <addaction name="actionRight"/>
    <addaction name="actionCenter"/>
    <addaction name="actionJustify"/>
    <addaction name="separator"/>
    <addaction name="actionFont"/>
    <addaction name="actionColor"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuFormat"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="separator"/>
  </widget>
  <action name="actionNew">
   <property name="icon">
    <iconset>
     <normaloff>:/src/images/new.png</normaloff>:/src/images/new.png</iconset>
   </property>
   <property name="text">
    <string>New</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="icon">
    <iconset>
     <normaloff>:/src/images/open.png</normaloff>:/src/images/open.png</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset>
     <normaloff>:/src/images/save.png</normaloff>:/src/images/save.png</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSave_as">
   <property name="text">
    <string>Save as</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset>
     <normaloff>:/src/images/exit.png</normaloff>:/src/images/exit.png</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="icon">
    <iconset>
     <normaloff>:/images/undo.png</normaloff>:/images/undo.png</iconset>
   </property>
   <property name="text">
    <string>Undo</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="icon">
    <iconset>
     <normaloff>:/images/redo.png</normaloff>:/images/redo.png</iconset>
   </property>
   <property name="text">
    <string>Redo</string>
   </property>
  </action>
  <action name="actionCut">
   <property name="icon">
    <iconset>
     <normaloff>:/images/cut.png</normaloff>:/images/cut.png</iconset>
   </property>
   <property name="text">
    <string>Cut</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="icon">
    <iconset>
     <normaloff>:/images/copy.png</normaloff>:/images/copy.png</iconset>
   </property>
   <property name="text">
    <string>Copy</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="icon">
    <iconset>
     <normaloff>:/images/paste.png</normaloff>:/images/paste.png</iconset>
   </property>
   <property name="text">
    <string>Paste</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="actionBold">
   <property name="icon">
    <iconset>
     <normaloff>:/images/bold.png</normaloff>:/images/bold.png</iconset>
   </property>
   <property name="text">
    <string>Bold</string>
   </property>
  </action>
  <action name="actionItalic">
   <property name="icon">
    <iconset>
     <normaloff>:/images/italic.png</normaloff>:/images/italic.png</iconset>
   </property>
   <property name="text">
    <string>Italic</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset>
     <normaloff>:/images/about.png</normaloff>:/images/about.png</iconset>
   </property>
   <property name="text">
    <string>about</string>
   </property>
  </action>
  <action name="actionPrint">
   <property name="icon">
    <iconset>
     <normaloff>:/images/right.png</normaloff>:/images/right.png</iconset>
   </property>
   <property name="text">
    <string>Print</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="actionPrint_Preview">
   <property name="icon">
    <iconset>
     <normaloff>:/images/printprev.png</normaloff>:/images/printprev.png</iconset>
   </property>
   <property name="text">
    <string>Print Preview</string>
   </property>
  </action>
  <action name="actionExport_PDF">
   <property name="icon">
    <iconset>
     <normaloff>:/images/pdf.png</normaloff>:/images/pdf.png</iconset>
   </property>
   <property name="text">
    <string>Export PDF</string>
   </property>
  </action>
  <action name="actionUnderline">
   <property name="icon">
    <iconset>
     <normaloff>:/images/underline.png</normaloff>:/images/underline.png</iconset>
   </property>
   <property name="text">
    <string>Underline</string>
   </property>
  </action>
  <action name="actionLeft">
   <property name="icon">
    <iconset>
     <normaloff>:/images/left.png</normaloff>:/images/left.png</iconset>
   </property>
   <property name="text">
    <string>Left</string>
   </property>
  </action>
  <action name="actionRight">
   <property name="icon">
    <iconset>
     <normaloff>:/images/right.png</normaloff>:/images/right.png</iconset>
   </property>
   <property name="text">
    <string>Right</string>
   </property>
  </action>
  <action name="actionCenter">
   <property name="icon">
    <iconset>
     <normaloff>:/images/center.png</normaloff>:/images/center.png</iconset>
   </property>
   <property name="text">
    <string>Center</string>
   </property>
  </action>
  <action name="actionJustify">
   <property name="icon">
    <iconset>
     <normaloff>:/images/justify.png</normaloff>:/images/justify.png</iconset>
   </property>
   <property name="text">
    <string>Justify</string>
   </property>
  </action>
  <action name="actionFont">
   <property name="icon">
    <iconset>
     <normaloff>:/images/font.png</normaloff>:/images/font.png</iconset>
   </property>
   <property name="text">
    <string>Font</string>
   </property>
  </action>
  <action name="actionColor">
   <property name="icon">
    <iconset>
     <normaloff>:/images/color.png</normaloff>:/images/color.png</iconset>
   </property>
   <property name="text">
    <string>Color</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
